react-search/Search/src/Search.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { SearchBar } from './SearchBar';
+import { List } from './List';
+
+const quotes = [
```

Comment:
To mimic how a Search component would work in a real app, these quotes should be moved to App.tsx and passed as a prop to Search.